web/src/main/java/com/futurewei/alcor/web/entity/dataplane/GoalStateJsonNB.java

```diff
@@ -0,0 +1,94 @@
+package com.futurewei.alcor.web.entity.dataplane;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.futurewei.alcor.common.constants.Common;
+import com.futurewei.alcor.common.entity.CustomerResource;
+import com.futurewei.alcor.web.entity.securitygroup.SecurityGroupEntity;
+import com.futurewei.alcor.web.entity.subnet.SubnetEntity;
+import com.futurewei.alcor.web.entity.vpc.VpcEntity;
+
+public class GoalStateJsonNB extends CustomerResource {
+
+  private Common.ResourceType rsType;
```

Comment:
I think both of ResourceType and OperationType belong to GoalStateJsonSB between DPM and ACA, so we could safely remove them from this class. 

We could use the REST Method (GET, POST, PUT, and DELETE) to determine the operation type.